src/controllers/projectController.js

```diff
@@ -27,4 +28,22 @@ module.exports = {
       return res.status(500).json({ message: 'Erro ao cadastrar projeto' });
     }
   },
+  async editProject(req, res) {
+    const projectToCheck = await projectModel.verifyId(
+      'projects',
+      'project_id',
+      req.body.projectId
+    );
+    if (projectToCheck === true) {
```

Comment:
Uma condição já avalia se o resultado é `true`, então essa verificação `=== true` é desnecessária

```suggestion
    if (projectToCheck) {
```